internal/resource/default_user_secret.go

```diff
@@ -146,6 +147,18 @@ func (builder *DefaultUserSecretBuilder) updatePorts(secret *corev1.Secret) {
 	}
 }
 
+// generateUsername returns a base64 string that has "default_user_" as prefix
+// returned string has length 'l' when base64 decoded
+func generateUsername(l int) (string, error) {
+	encodedStr, err := randomEncodedString(l)
+	if err != nil {
+		return "", err
+	}
+
+	encodedStrSlice := []byte(encodedStr)
```

Comment:
We could rename the variable `encodedStrSlice` since it's a byte slice, not a string slice.